packages/core/package.json

```diff
@@ -102,5 +102,9 @@
         "infile": "../../CHANGELOG.md"
       }
     }
+  },
+  "dependencies": {
+    "@dotlottie/react-player": "^1.4.6",
+    "@lottiefiles/react-lottie-player": "^3.5.3"
```

Comment:
1. Can we move these dependencies to the top (directly above `peerDependencies`)?
2. Are these dependencies always required? Just wondering if we can make them `peerDependencies` but the answer is probably no.

Reply:
I moved those libraries to `devDependencies` and optional `peerDependencies`. 

In that way, if someone needs only a native version, he doesn't need to install the web libraries (and increase bundle size). But the app can be still built (because it is in `devDependencies`).

I'm not sure if this is the correct approach, but it makes sense to me.